trailblazer/apps/slurm/utils/formatters.py

```diff
@@ -0,0 +1,46 @@
+from typing import Dict, Callable
+
+from trailblazer.constants import Pipeline
+
+
+def reformat_mip_job_step(job_step: str) -> str:
+    """Reformat data analysis MIP job step.
+    Raise: Exception if unable to split string.
+    """
+    try:
+        return "_".join(job_step.split("_")[:-1])
+    except Exception:
+        return job_step
```

Comment:
I think these are silent, is this intended or should the exception be raised?

Reply:
In general it is not great to catch `Exception` because we might end up catching and handling an exception that we had not considered and which should be fatal. Might be fine here though